storage/gcloud/aio/storage/blob.py

```diff
@@ -26,3 +38,75 @@ def chunk_size(self) -> int:
 
         self.__dict__.update(metadata)
         return metadata
+
+    async def get_signed_url(  # pylint: disable=too-many-locals
+            self, expiration: int, headers: Optional[dict] = None,
+            query_params: Optional[dict] = None, http_method: str = 'GET',
+            iam_client: Optional[IamClient] = None,
+            service_account_email: Optional[str] = None,
+            project: Optional[str] = None, service_file: Optional[str] = None,
+            token: Optional[Token] = None,
+            session: Optional[aiohttp.ClientSession] = None) -> str:
+        """ Create a temporary access URL for Storage Blob accessible by
+        anyone with the link.
+
+        # Adapted from Google Documentation:
+        https://cloud.google.com/storage/docs/access-control/signing-urls-manually#python-sample
```

Comment:
Looks like a pretty direct port to me. Only thing I saw missing was the `expiration > 604800` check. What's the reason behind removing it? I could go either way.

Reply:
nope, I can add that in if you want.

Reply:
Yeah, sounds good to keep the block as a direct port, then.